app/javascript/views/payments/Payments.tsx

```diff
@@ -1,28 +1,109 @@
 import * as React from "react";
 
 import { Container, Wrapper } from "../style";
-import { Card, Layout, Row } from "./paymentsStyle";
+import { Card, Layout, Row } from "./PaymentsStyle";
 
 import PaymentHistory from "./paymentHistory/PaymentHistory";
 import PaymentOverview from "./paymentOverview/PaymentOverview";
+import PaymentHeader from "./paymentsHeader/PaymentsHeader";
 import UpholdCard from "./upholdCard/UpholdCard";
 
-export default class Payments extends React.Component {
+import routes from "../routes";
+
+export interface IInvoice {
+  id: string;
+  date: string;
+  url: string;
+  finalized_amount: string;
```

Comment:
Is this supposed to be in camelCase or snake_case?

Reply:
Could work both ways

https://github.com/brave-intl/publishers/pull/1620/commits/ebcdf99c43312dfa555ca160238f83aed56b497c